src/components/AddBooks.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const AddBooks = () => (
+  <div>
+    <h2>ADD NEW BOOK</h2>
+    <form>
+      <input type="text" placeholder="Book Title" />
+      <input type="text" placeholder="Category" />
```

Comment:
- [ ] Kindly note that the form should take the *author*, book title and or category data, here we have category and book title. 😃